apps/nova/src/pages/UpcomingLessons.tsx

```diff
@@ -21,10 +55,35 @@ const UpcomingLessons: React.FC = () => {
     canceled: true,
   });
 
+  // useMutation hooks
+  const { mutate: createRating } = useCreateRatingTutor({
+    onSuccess: () => setRateDialogInfo((prev) => ({ ...prev, enabled: false })),
+    onError: () => console.log("createRating failed!"),
+  });
+
+  const { mutate: editRating } = useEditRatingTutor({
+    onSuccess: () => setRateDialogInfo((prev) => ({ ...prev, enabled: false })),
+    onError: () => console.log("editRating failed!"),
+  });
```

Comment:
You will not edit the rating, He will only "create" new rating if doesn't have yet.